api/server/router/container/container_routes.go

```diff
@@ -425,6 +425,9 @@ func (s *containerRouter) postContainerUpdate(ctx context.Context, w http.Respon
 	if err := decoder.Decode(&updateConfig); err != nil {
 		return err
 	}
+	if versions.LessThan(httputils.VersionFromContext(ctx), "1.40") {
+		updateConfig.PidsLimit = nil
```

Comment:
Wondering what is the "idiomatic" way to set properties of embedded structs? Is this better?

```suggestion
		updateConfig.Resources.PidsLimit = nil
```

Reply:
Hum, I tends to prefer the explicit way (aka `updateConfig.Resources.PidsLimit`) but not sure what is the "idiomatic" way :sweat_smile: